ui/lib/apps/Statement/pages/Detail/PlanDetailTabCopr.tsx

```diff
@@ -46,3 +50,13 @@ function genShortValueTooltipValueItem(
     value: <ShortValueWithTooltip value={Number(data[key])} />,
   }
 }
+
+function genScaledBytesTooltipValueItem(
```

Comment:
I suspect whether `genShortValueTooltipValueItem` and `genScaledBytesTooltipValueItem` is necessary :) Seems that it introduces new concepts / abstractions without reducing code duplicates.

Reply:
Yes, maybe I should resolve the todo I left here before. `// TODO: refactor items code gen for all DetailsList`
There's second time I meet the code.